UnityProject/Assets/Scripts/Tilemaps/Tiles/BasicTile.cs

```diff
@@ -34,6 +34,10 @@ public abstract class BasicTile : LayerTile
 	[Tooltip("Can this tile be mined?")] [FormerlySerializedAs("Mineable")] [SerializeField]
 	private bool mineable = false;
 
+
+	[Tooltip("RadiationPassability 0 = 100% Resistant")] [SerializeField]
+	public float RadiationPassability = 1;
```

Comment:
should the range be from 0 to 1 then?

Reply:
humm You can't have a range for floats sadly has to be int?

Reply:
https://docs.unity3d.com/ScriptReference/RangeAttribute.html according to this you can

Reply:
must have forgot to included the float value 0,1 doesn't work unless you have 0f, 1f